web/src/lib/components/assets/thumbnail/image-thumbnail.svelte

```diff
@@ -1,27 +1,38 @@
 <script lang="ts">
-	import { imageLoad } from '$lib/utils/image-load';
+	import { onMount } from 'svelte';
+	import { lazyLoad } from 'unlazy';
+	import * as Buffer from 'buffer';
 
 	export let url: string;
 	export let altText: string;
 	export let heightStyle: string | undefined = undefined;
 	export let widthStyle: string;
+	export let thumbhash: Buffer;
 	export let curve = false;
 	export let shadow = false;
 	export let circle = false;
-	let loading = true;
-</script>
-
-<img
+  
+	let imageElement: HTMLImageElement;
+  
+	onMount(() => {
+		const Str = Buffer.Buffer.from(thumbhash).toString('base64');
```

Comment:
Importing the whole Buffer functionality is a bit overkill when you only want to convert from base64 to byte array

Reply:
This is gone now as the conversion has been moved to the server.